nucypher/policy/collections.py

```diff
@@ -76,7 +78,11 @@ def __init__(self,
                  destinations=None,
                  message_kit: UmbralMessageKit = None,
                  public_signature: Signature = None,
-                 hrac: Optional[bytes] = None) -> None:
+                 hrac: Optional[bytes] = None,
+                 version: bytes = None) -> None:
```

Comment:
In what cases would we want to specify the version externally? A version determines how an object is serialized to bytes, which is hardcoded into the class definition.